Mapsahoy.. get it.. like the cereal chips ahoy.. haha..

DoctorOctopusTheOctopi opened the pull request:
Commit #1
- Fixes/Updates Niflheim
- Fixes/Updates CMV_Elivagar
- Purges the remnants of Nanotrasen (probably.)
- Removes EGUNs from all lockers.
- Adds a Cryobay to the Exoplanet Research upper-levels.
- Removes an OP Chemistry Dispenser, replaces it with a regular one while revamping the Medical Bay.

To:Do:
- Add Niflheim Jobs
- Add Niflheim Overmap
- Purge Mining Colony
- Fix Covvie vessel.
- (maybe) Re-do Ice planet.
- probably other stuff i'll forget.

DoctorOctopusTheOctopi committed:
Local Repo update. (#1)

* deck gun value modifications

* shield hit-through-locs removed

* kig-yar pirate icon

* deck gun changes

* kig-yar shield buff

* needler ammo change

* overmap weapon sound modifications

* missile pods

* pulse turrets

* removes debug area

* overmap weapon implementation prep

* Map fixes.

CHANGES:
Exoresearch 1 and 2:
     *Added Cryodorms to lower level of research base.
     *Labelled all areas in the research base for easier teleporting.
     *Readded Sharpshooter gear to ODST vendor.
     *Added ODST Sharpshooter and ODST Combat Engineer als alternate job titles.

CCV_Star:
     *Added Connector port for refueling, spare fuel added.
     *Fixed an engine piping issue.

CCV_Slow_But_Steady:
     *Added voidsuits to CCV_Slow_But_Steady so its crew can reach the airlock on the other side of the vessel.
     *Added one Civ Ship Crew spawnpoint to the CCV_Slow_But_Steady. Having to play alone is sad.

* Cryodorms.

KS7-535
*Added A cryodorms room.

Asteroid Listening Post.
*Added A cryodorms room.

* areas

* corvette rocket array

* nerfs pulse turret damage

* invis nerf

* landmine nerf

* sector-hit effects

* Adds a mining asteroid colony.

Adds PYK-248, a mining colony based on an asteroid.
*Spawnpoints for 10 colonists.
*Medical Bay, mining equipment, living quarters, sushi bar, cryodorms all added
*4 docking umbilicals at the edge of the facility.

* Adds map marker and overmap Icon to Mining asteroid

*Adds map marker and overmap Icon to mining asteroid.

* umbilical object movement

* Various Quality Passes on maps.

UNSC Corvette:
Redesigned the corvette to accomodate cryodorms
Added signs to the corvette
Set acces lists on some doors
added 2 holding cells
relocated medbay and enlarged it
added janitorial closet
added supply room
added captain quarters.
added separate spawnpoint for corvette captain.

CCV Comet
Added drugs to the URF ship.
Added Cryodorms to the URF ship.

* Adds BR85 battle rifle to ODST vendor

BR85 added to ODST vendor (2)
M95_sap ammo added to ODST vendor (10)

* Spawn point fix.

Fixes the colonist spawnpoint.

* Replaced matter bins with regular bins.

Replaced matter bins with regular bins on all maps that had them.

* Map Fixes, overmap debug.

Fixes the overmap, icon placement, prevents planets spawning as each other
Fixes duplicate planets on overmap
Fixes invisible kig-yar turrets

* Fixes space turfs on mining outpost

Fixes some turfs I missed under airlocks on mining outpost.

* Space turf fix

Fixed for realsies now.

* Bin tweak. last commit

Bin tweak. last commit

* [ci skip] Update Changelogs

* buff explosion resist and integrity

* projresist window ex_act buff

* Fixed walls, added blood packs.

Added blood pack crates to medical bays, fixed an issue with wall icons.
Fixes Depot Guard job.

* adrenaline buff

* grunt spaceproofing

* base skirmisher stuff

* Geminus City Tweaks/ Theta Project Change (#612)

Geminus City Tweaks/ Theta Project Change

* Changed name of eyeballs to Augmented Eyeballs in theta room.
* Changed name of Enhanced Heart to Augmented Heart in theta room.
* Expanded the Theta area to accommodate a new closet.
* Added in a Project Theta Closet.
* Added in Theta Project Armor, Helmet, Arm Guards, boots and undersuit. (All unique items with custom descriptions- All not as good as Spartan armor)
* Added in Augmented Lungs. Unique item comparable to Augmented Liver.
* Changed Max damage for Augmented Eyes to 60 from 45 to be slightly tougher as it doesn't offer that much of a benefit other then flash resistance.
* Adding in three stacks of ten sheets of glass into cargo storage area.
* Adding in two stacks of ten sheets of glass into the Engineering Area of Geminus.
* Added a Sink to the Padded Cell in Hospital.
* Added additional sink closer to Surgery table at GCPD Morgue.
* Moved Microwave in hospital break room so people can access the Getmore Vendor.
* Added in recharger at hospital where spare Defibs are kept.
* Added in sink to hospital Morgue.
* Added holopads inside a number of buildings outside of the slums. (10-15 extra Mostly in government/public buildings)
* Added sink to Viro Bathroom.

* Asteroid mining fix

*Fixes incorrect space tiles on mining asteroid.

* Fixes docking umbilicals

Fixed some umbilical directions.

* adds base_turf to areas for various planet maps so that explosions wont vent to space

Signed-off-by: Cael_Aislinn <cael_aislinn@yahoo.com.au>

* Asteroid mining fix (#619)

* Asteroid mining fix

*Fixes incorrect space tiles on mining asteroid.

* Fixes docking umbilicals

Fixed some umbilical directions.

* Working on airlocks, replicator tweaks.

*Working on functional airlocks on all maps.
*Tweaked replicator coding.

* removable shrapnel

* working on ODST updates

Working on ODST AND ONI SEC GUARD switch, coding complete.

* Finished switch from ODST guards to ONI guards.

Added an ONI guard squad on research
removed ODST from research
Added ODST to corvette.

* Added airlocks

Airlocks added to PYK and asteroid2. requesting another mapper review to make them functional. (All the items are connected, but the airlock doesnt properly pump air out or in anymore.)
Added drop pod markers to PYK and KS7535

* Conflict fix

Attempting conflict fix.

* Manual remap based on current live map.

Hopefully this fixes the conflict.

* Research acces fixes, corvette QA pass.

*Minor changes to lights and a vendor on corvette as part of a QA pass.
*Fixes acces levels on airlocks for ONI guards.
*Fix for runtime error on asteroid listening post.

* Commit #1

- Adds eyeglasses to the loadout.
- Codes an ONI Badge

* Commit #2

- Sprited ONI Badge.
- Added ONI Badge to ONI Security Guards, and Researcher.

* Commit #3

- Resprite all Nanotrasen Items, changing them to Misriah Armory.
- Renamed most Nanotrasen Items, changing them to Misriah Armory.

* Bug Fixes?

* Revert "Bug Fixes?"

This reverts commit edc2172099b6863364f4ffbe999817438d1541c9.

* Finishing Touches

- Finishing Touches.

* Fixes #1, Its a long road boys.

* Final changes.

- Adds a head loadout spot.

* skirmisher implementation

* whitelist multi-rank fixes

* quill-name-fix

* Changelog

DoctorOctopusTheOctopi committed:
Revert "Local Repo update. (#1)" (#2)

This reverts commit f2a1a14a6fb54c853b380621c646e297d318a103.

DoctorOctopusTheOctopi committed:
Merge branch 'alpha' of https://github.com/DoctorOctopusTheOctopi/HaloSpaceStation13 into alpha

DoctorOctopusTheOctopi committed:
Commit #1

- Fixes/Updates Niflheim
- Fixes/Updates CMV_Elivagar
- Purges the remnants of Nanotrasen (probably.)
- Removes EGUNs from all lockers.
- Adds a Cryobay to the Exoplanet Research upper-levels.
- Removes an OP Chemistry Dispenser, replaces it with a regular one while revamping the Medical Bay.

bloxgate commented:
Please, please, please, start using descriptive commit names. Commit #1 is extremely lacking in any information.

DoctorOctopusTheOctopi committed:
Relocates Niflheim Files

- Puts them in their own folder.
- Removes the Niflheim Areas from the om_ship_areas

DoctorOctopusTheOctopi committed:
Fixes ONI guard loadout, fixes additional minor bug, additional tweaks.

- Fixes the ONI Badge not being able to transform into an ONI Badge again.
- Adds 311 access to airtight airlocks
- Fixes a bug with ONI Jobs.

DoctorOctopusTheOctopi committed:
appeases travis

- Fixes the Travis error involving areas.

DoctorOctopusTheOctopi committed:
Fixes Travis Error, Messes with ExoResearch, Defines Jobs,

- Fixes the Travis Error
- Adds a new cryobay, elevator, and remodeled medbay to ExoResearch, -
- Defines Niflheim's Jobs.

DoctorOctopusTheOctopi commented:
I've decided to shutdown this project mostly because I've lost passion for it and its going to conflict the shit out of things for a shitty prison I made. I'll be making better stuff in the future.

caelaislinn commented:
The code changes looked ok. It's generally a bad idea to bundle map tweaks for multiple maps together and then put in code changes as well.
